src/Http/Controllers/Admin/TranslationsController.php

```diff
@@ -67,4 +73,196 @@ public function update(UpdateTranslation $request, Translation $translation)
+            } else {
+                $collection = $collection->map(function ($row) use ($request, $existingTranslations, $chooseLanguage) {
+                    if(!$this->rowValueEqualsValueInArray($row, $existingTranslations, $request)){
+                        $row['has_conflict'] = true;
+                        if(isset($existingTranslations[$this->buildKeyForArray($row)])){
+                            $row['current_value'] = strval($existingTranslations[$this->buildKeyForArray($row)]['text'][$chooseLanguage]);
+                        } else {
+                            $row['current_value'] = "";
+                        }
+                        return $row;
+                    }
+                    $row['has_conflict'] = false;
+                    return $row;
+                });
+
+                $conflicts = $collection->filter(function($row){
+                    return $row['has_conflict'];
+                })->count();
+
+                if($conflicts == 0){
+                    $numberOfUpdatedTranslations = 0;
+                    $numberOfImportedTranslations = 0;
+
+                    $collection->map(function($item) use ($chooseLanguage, $existingTranslations, &$numberOfImportedTranslations, &$numberOfUpdatedTranslations){
+                        if(isset($existingTranslations[$this->buildKeyForArray($item)]['id'])){
+                            $id = $existingTranslations[$this->buildKeyForArray($item)]['id'];
+                            $t = Translation::find($id);
+                            $textArray = $t->text;
+                            if($textArray[$chooseLanguage] != $item[$chooseLanguage]){
+                                $numberOfUpdatedTranslations++;
+                                $textArray[$chooseLanguage] = $item[$chooseLanguage];
+                                $t->update(['text' => $textArray]);
+                            }
+                        } else {
+                            $numberOfImportedTranslations++;
+                            $this->createOrUpdate($item['namespace'], $item['group'], $item['default'], $chooseLanguage, $item[$chooseLanguage]);
+                        }
+                    });
+
+                    return ['numberOfImportedTranslations' => $numberOfImportedTranslations, 'numberOfUpdatedTranslations' => $numberOfUpdatedTranslations];
+                } else {
+                    // let user decide how to resolve conflicts
+                    return $collection;
+                }
+            }
+
+        }
+    }
+
+    private function buildKeyForArray($row)
+    {
+        return $row['namespace'] . '.' . $row['group'] . '.' . $row['default'];
+    }
+
+    private function rowExistsInArray($row, $array)
+    {
+        return array_key_exists($this->buildKeyForArray($row), $array);
+    }
+
+    private function rowValueEqualsValueInArray($row, $array, $request)
```

Comment:
You are missing types and return values in methods, the package require php 7.1 so you can use them.